xradar/io/export.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+# Copyright (c) 2022, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+"""
+
+ODIM_H5 output
+==============
+
+This sub-module contains the writer for export of ODIM_H5-based radar
+data into.
+
+Code ported from wradlib.
+
+Example::
+
+    import xradar as xd
+    dtree = xd.io.to_odim(dtree, filename)
+
+.. autosummary::
+   :nosignatures:
+   :toctree: generated/
+
+   {}
+
+"""
+
+__all__ = [
+    "to_odim",
+]
+
+__doc__ = __doc__.format("\n   ".join(__all__))
+
+import datetime as dt
+
+import h5py
+import numpy as np
+
+from ..model import required_sweep_metadata_vars
+
+
+def _write_odim(src, dest):
```

Comment:
Can you write out source and destination here?

Reply:
@mgrover1 Yes, Let me add this here

Reply:
done